services/gitea/4.1.1/defaults/cm.yaml

```diff
@@ -9,11 +9,22 @@ data:
     ---
     image:
       rootless: true
+    ingress:
+      enabled: true
+      annotations:
+        kubernetes.io/ingress.class: kommander-traefik
+        traefik.ingress.kubernetes.io/router.middlewares: kommander-stripprefixes@kubernetescrd
+        traefik.ingress.kubernetes.io/router.tls: "true"
+      hosts:
+        - paths:
+          - path: /dkp/kommander/gitserver
```

Comment:
what about just `/dkp/kommander/git`?

Reply:
I'm fine with that. `gitserver` is what's used in Kommander 2.0 but since the Git repo URL will change, anyway we can just as well use `git` here.

Reply:
Should it be added to traefik prefix overrides? (it resides in this repo and in kommander-cli as well - in the migration manifests).

Reply:
@mikolajb done